BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -338,4 +556,4 @@ public void onLongItemClicked(int position, ShelfItem shelfItem, View v) {
       selectedShelfItems.add(shelfItem);
     }
   }
-}
+}
```

Comment:
Please add a newline at the end of a file.
```suggestion
}

```

Reply:
This code is not written by me,, but okay, I will write a new line.